docs/doxygen/building.dox

```diff
@@ -107,6 +107,58 @@ Note: A pre-signed GET URL is required for all HTTP S3 demos. For upload demos,
 	- `--key` : The name of the existing object you wish to download (GET), or the name of the object you wish to upload (PUT).
 	- `--region` : The name of the region in which your S3 bucket was created. If this is not passed in, the region configured in AWS CLI (or us-east-1, if not configured) will be used.
 
+
+@section configuring_ota_demos Configuring the Over-The-Air Update Demos
+@brief Configurations and Prerequisites for the OTA Demo
+
+### Prerequisites:
+<ol>
+	<li> To perform a successful OTA update you would need to complete the Prerequisites mentioned here: https://docs.aws.amazon.com/freertos/latest/userguide/ota-prereqs.html </li>
+	<li> Code-Signing profile can be created using: https://docs.aws.amazon.com/freertos/latest/userguide/ota-code-sign-cert-esp.html </li>
+</ol>
+
+### Configuration:
+
+You will need to specify the following parameters in the file demo_config.h (located in `demos/ota_demo_[mqtt/http]/`)
+<ol>
+<li> AWS_IOT_ENDPOINT:
+     This is the endpoint for your account. The endpoint should be for the region that your thing was created in. This can be found by going to the iot core console, pressing the “Settings” tab on the left hand side of the page. It is under the section labeled “Custom endpoint”. </li>
+<li> CLIENT_CERT_PATH:
+    This is the client certificate that was downloaded when you created your thing. The path specified needs to be either an absolute path, or a path that is relative to the “cloned-repo-root-dir/build/bin" directory where the demo will be ran from.</li>
+<li> CLIENT_PRIVATE_KEY_PATH:
+    This is the client private key that was downloaded when you created your thing. The path specified needs to be either an absolute path, or a path that is relative to the “cloned-repo-root-dir/build/bin" directory where the demo will be ran from.</li>
+<li> CLIENT_IDENTIFER:
+    This is the name of the thing you created during the OTA Prerequisites section.</li>
+</ol>
+
+An example expected output:
+@code{sh}
+#define AWS_IOT_ENDPOINT           "endpoint##-ats.iot.us-west-2.amazonaws.com"
+#define CLIENT_CERT_PATH           "/home/ubuntu/certificates/thing#-certificate.pem.crt"
+#define CLIENT_PRIVATE_KEY_PATH    "/home/ubuntu/certificates/thing#-private.pem.key"
+#define CLIENT_IDENTIFIER          "testclient"
+@endcode
+
+### Scheduling an OTA Update Job:
+After you build and run the initial executable you will have to create another executable and schedule an OTA update job with this image.
+<ol>
+	<li> Increase the version of the application by setting \#define APP_VERSION_BUILD 3 in  demos/ota_demo_core_[mqtt/http]/demo_config.h</li>
+	<li> Rebuild the application using step 1 and 2 from above into a different directory, say build-dir-2</li>
```

Comment:
step 1 and 2?

Reply:
Thanks for the callout: updated to `@ref building_demo_commmandline`